CHANGELOG.md

```diff
@@ -6,6 +6,36 @@ separate changelogs for each crate were used. If you need to refer to these old
 
 ## [Unreleased]
 
+The main feature added here is support in `libcnb-test` for expanded testing on local buildpacks. Previously it
+was possible to test a single (local) buildpack with `BuildpackReference::Crate` for development testing as well as
+external (remote) buildpacks with `BuildpackReference::Other(String)`. This support has been expanded to allow 
+testing of any local buildpack (including meta-buildpacks) with `BuildpackReference::Local(PathBuf)`. This should make 
+developing and testing changes across multiple buildpacks easier on the buildpack author.
+
+### Added
+
+- `libcnb-package`
+  - Added `find_cargo_workspace` which provides a convenient starting point for locating buildpacks for packaging and testing purposes. ([#590](https://github.com/heroku/libcnb.rs/pull/590))
+  - Added the `BuildpackOutputDirectoryLocator` which contains information on how compiled buildpack directories are structured and provides a `.get(buildpack_id)` method which produces the output path for a buildpack. ([#590](https://github.com/heroku/libcnb.rs/pull/590))
+  - Added `output::assemble_single_buildpack_directory` and `output::assemble_meta_buildpack_directory` which construct buildpack output directories with all their required files during packaging. ([#590](https://github.com/heroku/libcnb.rs/pull/590))
+
+### Changed
+
+- `libcnb-package`
```

Comment:
> Are we treating `libcnb-package` as a "true" cargo project with version guarantees, or is it more like this internal serde crate

Good question. I'd thought the former, but I'm open to either really, as long as we're clear with potential external consumers of the create. Also at the moment with libcnb being pre-v1, we're bumping the major version most releases, so this mostly won't be an issue - and later once we go v1 I suspect `libcnb-package` will have settled down enough that we don't expect too much breakage.

One thing to watch out for though - if we were to switch to being more relaxed about the major version for `libcnb-package` - we'd really have to switch to using hard version pins instead of ranges for intra-crate dependencies. ie: `libcnb-cargo` would need to depend on `libcnb-package `==1.2.3` not `^1.2.3`. (We should perhaps make that switch regardless given our unified release versions approach.)

Reply:
@schneems Just to follow up here - we've now (a) switched to hard version pins in #644, (b) marked `libcnb-package` (and the new `libcnb-common`) as internal in their READMEs in #662.